src/modules/Packages.rb

```diff
@@ -1741,6 +1741,10 @@ def Initialize_StageInitial(show_popup, base_url, log_url)
         end
       end
 
+      # Lower the priority of the initial installation repository to prefer
+      # the packages from the other media (bsc#1071742)
```

Comment:
Does this cause in opensuse installation to prefer network over DVD when there are identical packages?

Reply:
Hmm... good point.

Then the DVD preference would not work correctly... :worried: 

We should do it only when *all* repos are DVD...

Reply:
Which means we cannot do it at this place as the addons are added later and we do not know in advance if all will be DVDs...